src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -3488,13 +3489,17 @@ public IEnumerable<BlittableJsonReaderObject> GetReplicationHubCertificateByHub(
 
                 if (pageSize-- < 0)
                     break;
+                
                 blittable.Modifications = new DynamicJsonValue(blittable)
                 {
-                    ["CertificateThumbprint"] = thumbprint,
+                    ["Certificate"] = GetCertificateAsBase64(val),
```

Comment:
Does it break compatibility with e.g. 5.0?
Why are we not using here nameof?

Reply:
added `nameof` and removed the `HubDefinitionName` & `Database` that are not used by Studio in this case